docs/xdr/features/collect/integrations/network/cisco_wsa.md

```diff
@@ -0,0 +1,12 @@
+uuid: 23b75d0c-2026-4d3e-b916-636c27ba4931
+name: Cisco Web Security Appliance
+type: intake
+
+## Overview
+
+The Cisco Web Security Appliance is a security device analyzing HTTP(S) traffic with malware detection and reputation filtering.
+  Sending Cisco Web Security Appliance logs to SEKOIA.IO will help you
+  discovering potential network security threats, which are contextualized 
+  thanks to our Cyber Threat Intelligence (CTI).
```

Comment:
Don’t use "I", "our", "us", "we", use "you".

https://zeroheight.com/6104b1d12/p/859e67-pronouns